elementary/monitor/data_monitoring/alerts/integrations/slack/message_builder.py

```diff
@@ -78,6 +78,12 @@ def add_details_to_slack_alert(
             ]
             self._add_blocks_as_attachments(configuration_blocks)
 
+    def add_color_to_slack_alert(self, color: str):
```

Comment:
Maybe we should move it to the generic `SlackMessageBuilder`?
It feels like adding color is a generic format that could be used in other places as well 🙂

Reply:
And we will need to rename the method 🙂

Reply:
Maybe add a comment for the color string format?
Just to make it easier for new ones 🙂